### ISSUE TITLE
When  countrySearch: false, and you search for a country it does not scroll to it

### ISSUE THREAD

**Issue by Nedimko123** [jackocnr/intl-tel-input#1514]
When you set countrySearch: false, and you search for a country it does not scroll to it

**Comment by Nedimko123** [comment]
For example the version:

```
import { component$, useSignal, useStore, useStyles$, useVisibleTask$ } from '@builder.io/qwik'

export default component$(() => {
    useVisibleTask$(() => {
        var input = document.querySelector('#phone')
        window.intlTelInput(input, {
            utilsScript: 'https://cdn.jsdelivr.net/npm/intl-tel-input@18.1.1/build/js/utils.js',
            formatOnDisplay: true,
            countrySearch: false,
        })
    })
    return (
        <>
            <input type="text" id="phone" class="bg-gray-700 text-white" />

            <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intl-tel-input@18.1.1/build/css/intlTelInput.css" />
            <script src="https://cdn.jsdelivr.net/npm/intl-tel-input@18.1.1/build/js/intlTelInput.min.js"></script>
        </>
    )
})
```


Has the working search but It does not format the numbers, so what can I do?

**Comment by Nedimko123** [comment]
Version 19.1.1 is when I notice that it does format the number on input

**Comment by jackocnr** [comment]
Thanks for reporting. Fix released in v19.2.7.
